src/Generating/StructGen.cpp

```diff
@@ -183,6 +183,14 @@ void cStructGenTrees::ApplyTreeImage(
 				CASE_TREE_OVERWRITTEN_BLOCKS:
```

Comment:
I'd just like to take a moment to appreciate the peculiarity of this statement. It is truly a *case* of unnecessary macro use.